### DISCUSSION TITLE
ZTnet migration

### DISCUSSION THREAD

**Discussion by marziniconsulting** [sinamics/ztnet#615] [General]
Hello,
I've installed ZTnet on a Google Cloud VM machine and evertyhing works perfect except sometime I've to reboot because the machine crash due low memory.
Anyway, I would like to migrate the installation to a different VM on a different provider with a different ip address. I want to maintain all the configured machine with zerotier and avoid to create new networks etc.
Is it possible? Thank you!

Best,
Alessandro

**Comment by PizzaProgram** [comment]
I guess it depends, whether you run it as:
- self-hosted root planet, 
- or default moon?

In either cases, I think the change of the IP address is a problem.

**Comment by marziniconsulting** [comment]
self hosted root planet that I think is the default after the installation

**Comment by PizzaProgram** [comment]
No, default is a **moon**. Part of the Official ZT network.

You have to click "Create self hosted planet" (or whatever the button is called, do not remember exactly any more,) to have your own planet.

So, If you have "default", it's "out of ZTnet's hand", whether the clients will be re-recongnised or not. 

### I'm not an expert, I only know that:
- If you have self-hosted, you can have 2+ servers, set before.
- But than you have to change all the planet files, so it's lot of trouble. (Also Mobil platform limitation!)

## So I'm interested too: 
 - How can someone backup the whole system properly?

I guess it's easier to leave it as it is now, but how the clients will find the new (moon-) server automatically, is a mistery for me too.

**Reply by PizzaProgram** [reply]
There is a similar problem described here: 
https://github.com/sinamics/ztnet/discussions/559
So I guess it will be problematic.
